stylelint-polaris/index.js

```diff
@@ -198,6 +220,14 @@ module.exports = {
           /([\w-]+\.)?rem/,
           /([\w-]+\.)?spacing/,
         ],
+        'declaration-property-unit-disallowed-list': [
+          {
+            '/^padding/': ['px', 'rem', 'em'],
+            '/^margin/': ['px', 'rem', 'em'],
+            '/^gap/': ['px', 'rem', 'em'],
+          },
+          {severity: 'warning'},
```

Comment:
I do wonder if this should be a failure 🤔. We might want to review warning/failures as a group. Not blocking for this PR.

Reply:
💯